libs/native-federation/src/utils/updateIndexHtml.ts

```diff
@@ -51,5 +51,14 @@ export function updateScriptTags(
   );
   indexContent = indexContent.replace(/<script src="main.*?><\/script>/, '');
   indexContent = indexContent.replace('</body>', `${htmlFragment}</body>`);
+
+  // add link to importmap.json
+  const scriptTagImportMap =
```

Comment:
#### Do not process host during `initFederation`
The **Host** `importmap.json` is directly included in `index.html`. No need to process it anymore.